src/mobile/src/ui/components/TransferConfirmationModal.js

```diff
@@ -121,86 +119,110 @@ class TransferConfirmationModal extends Component {
+                                >
+                                    <Text style={[styles.messageText, { color: body.color }]}>{message}</Text>
+                                </View>
+                            </ScrollView>
+                        )}
+                    {!isMessage && (
+                        <View style={{ paddingTop: height / 80, alignItems: 'center' }}>
+                            <View style={styles.valueContainer}>
+                                <TextWithLetterSpacing
+                                    spacing={width / 100}
+                                    textStyle={[styles.valueTextLarge, { color: body.color }]}
+                                >
+                                    {round(formatValue(value), 3)}
+                                </TextWithLetterSpacing>
+                                <Text
+                                    style={[
+                                        styles.valueTextSmall,
+                                        { color: body.color, marginTop: height / 36, paddingLeft: width / 40 },
+                                    ]}
+                                >
+                                    {formatUnit(value)}
                                 </Text>
-                            </Text>
-                            <Text style={[styles.regularText, textColor, { paddingTop: height / 90 }]}>to</Text>
-                            <Text style={[styles.addressText, textColor, { marginTop: height / 70 }]}>
-                                {this.props.address.substring(0, 30)}
-                            </Text>
-                            <Text style={[styles.addressText, textColor]}>{this.props.address.substring(30, 60)}</Text>
-                            <Text style={[styles.addressText, textColor, { marginBottom: height / 18 }]}>
-                                {this.props.address.substring(60, 90)}
+                            </View>
+                            <Text style={[styles.valueTextSmall, { color: body.color, marginTop: height / 200 }]}>
+                                {conversionText}
                             </Text>
                         </View>
                     )}
-                    <ModalButtons
-                        onLeftButtonPress={() => this.props.hideModal()}
-                        onRightButtonPress={() => this.onSendPress()}
-                        leftText={t('global:cancel').toUpperCase()}
-                        rightText={t('global:send')}
-                        buttonWidth={{ width: width / 3.2 }}
-                        containerWidth={{ width: width / 1.4 }}
+                </View>
+                <View style={{ paddingVertical: height / 40 }}>
+                    <LottieView
+                        source={arrowAnimation}
+                        style={{ width: width / 17, height: height / 18 }}
+                        loop
+                        autoplay
```

Comment:
```suggestion
                        autoPlay
```
For whatever reason it's `autoplay` in https://github.com/airbnb/lottie-web but `autoPlay` in https://github.com/react-community/lottie-react-native#api